currency_rate_update_nbu/data/res_currency_rate_provider_data.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Why do you need this file? Also, I would guess if you do want to auto-create the provider, it needs to be `noupdate`